tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
We allow up to 119 characters as this is the width of GitHub code review, anything longer requires horizontal scrolling which makes review more difficult. Documentation, comments, and docstrings should be wrapped at 79 characters.

xD